tutorials/eda_contents/numpy/numpy.ipynb

```diff
@@ -0,0 +1,1408 @@
+      " -3.68000000e+00 -3.60000000e+00 -3.52000000e+00 -3.44000000e+00\n",
+      " -3.36000000e+00 -3.28000000e+00 -3.20000000e+00 -3.12000000e+00\n",
+      " -3.04000000e+00 -2.96000000e+00 -2.88000000e+00 -2.80000000e+00\n",
+      " -2.72000000e+00 -2.64000000e+00 -2.56000000e+00 -2.48000000e+00\n",
+      " -2.40000000e+00 -2.32000000e+00 -2.24000000e+00 -2.16000000e+00\n",
+      " -2.08000000e+00 -2.00000000e+00 -1.92000000e+00 -1.84000000e+00\n",
+      " -1.76000000e+00 -1.68000000e+00 -1.60000000e+00 -1.52000000e+00\n",
+      " -1.44000000e+00 -1.36000000e+00 -1.28000000e+00 -1.20000000e+00\n",
+      " -1.12000000e+00 -1.04000000e+00 -9.60000000e-01 -8.80000000e-01\n",
+      " -8.00000000e-01 -7.20000000e-01 -6.40000000e-01 -5.60000000e-01\n",
+      " -4.80000000e-01 -4.00000000e-01 -3.20000000e-01 -2.40000000e-01\n",
+      " -1.60000000e-01 -8.00000000e-02  3.55271368e-15  8.00000000e-02\n",
+      "  1.60000000e-01  2.40000000e-01  3.20000000e-01  4.00000000e-01\n",
+      "  4.80000000e-01  5.60000000e-01  6.40000000e-01  7.20000000e-01\n",
+      "  8.00000000e-01  8.80000000e-01  9.60000000e-01  1.04000000e+00\n",
+      "  1.12000000e+00  1.20000000e+00  1.28000000e+00  1.36000000e+00\n",
+      "  1.44000000e+00  1.52000000e+00  1.60000000e+00  1.68000000e+00\n",
+      "  1.76000000e+00  1.84000000e+00  1.92000000e+00  2.00000000e+00\n",
+      "  2.08000000e+00  2.16000000e+00  2.24000000e+00  2.32000000e+00\n",
+      "  2.40000000e+00  2.48000000e+00  2.56000000e+00  2.64000000e+00\n",
+      "  2.72000000e+00  2.80000000e+00  2.88000000e+00  2.96000000e+00\n",
+      "  3.04000000e+00  3.12000000e+00  3.20000000e+00  3.28000000e+00\n",
+      "  3.36000000e+00  3.44000000e+00  3.52000000e+00  3.60000000e+00\n",
+      "  3.68000000e+00  3.76000000e+00  3.84000000e+00  3.92000000e+00]\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(f\"The y array length is {len(y)} and values \\n {y}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Explanation\n",
+    "This method has created an array of length $100$ with an interval of $[-4,4)$. For more information, pleast take a look at the [arange documentation](https://numpy.org/doc/stable/reference/generated/numpy.arange.html)\n",
+    "<br>\n",
+    "<br>\n",
+    "<br>\n",
+    "<br>\n",
+    "<br>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Visualising each 2D array using `pcolormesh`\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### meshgrid()\n",
+    "\n",
+    "This function is used to create a rectangular grid out of two $1D$ arrays. This function operates by making one array a $n \\times 1$ array and the other $1 \\times n$ and returning a a matrix corresponding to their interaction.\n",
+    "\n",
+    "<img title=\"meshgrid visualisation\" src=\"numpy_mesh_grid.png\">"
```

Comment:
This image is not rendered. It may mean I have to change some settings on the _config.yml file

Reply:
let me know if i need to make any changes on my end